### DISCUSSION TITLE
Is it possible to add simple workflows?

### DISCUSSION THREAD

**Discussion by falbalae** [papermerge/papermerge-core#351] [Ideas]
Hello @ciur and compliments for the nice work. 
From my understanding it is not possible yet to ask Papermerge to save documents in specific folders based on what the OCR reads. The classic example: I have the bill document from my phone company. I would like papermerge to understand based on some information which are written on the PDF file that the document is the bill of the phone company. 

It would be even better if I could ask Papermerge to look for specific information and fill customs fields by itself, but here might be complicated. For example it would be nice if in the aforementioned bill papermerge could read the date of the bill, the amount of the bill and insert it in fields that I would have created, one for the date and one for the amount. 

But for now I would already be happy if I could tell Papermerge that every PDF file which contains the name of phone company has to go in a specific folder. 

Once more thanks and keep it up! :-)

**Comment by ciur** [comment]
For the moment, you can use REST API for that. In case you know Python, you can use requests library to play with REST API. But if you are master of bash, even curl will do.

Workflows are on the roadmap but will come much later, as the basic features are not there yet (like  e.g. custom fields, user options)
